crates/burn-core/src/module/param/primitive.rs

```diff
@@ -52,6 +57,20 @@ where
     }
 }
 
+impl<T: ModuleDisplay> ModuleDisplayDefault for Option<T> {
+    fn content(&self, content: Content) -> Option<Content> {
+        match self {
+            Some(module) => content.add_single(module).optional(),
+            None => {
+                let none_string = "None".to_string();
+                content.add_single(&none_string).optional()
```

Comment:
could be `content.add_single("None").optional()` I think?

Reply:
I get this error:

```rust
error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> /Users/dilshod/Projects/burn/crates/burn-core/src/module/param/primitive.rs:64:40
    |
64  |             None => content.add_single("None").optional(),
    |                             ---------- ^^^^^^ doesn't have a size known at compile-time
    |                             |
    |                             required by a bound introduced by this call
    |
```

But I fixed it with this new signature:

```rust
    pub fn add_single<T: ModuleDisplay + ?Sized>(mut self, value: &T) -> Self {
```

Now, it works.